site/src/atomics/aspect-ratio.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Aspect ratio
+description: Aspect ratio related atomic css classes for the Atlas Design System
+template: standard
+classType: Atomics
+classPrefixes:
+  - aspect-ratio
+---
+
+# Aspect Ratio Atomics
+
+Ratio atomics are used to add a fluid aspect ratio to an element. The element will scale to fill 100% width of the container while matching the height to the defined aspect ratio.
+
+| cssproperty  | ratio                                             | screensize |
+| ------------ | ------------------------------------------------- | ---------- |
+| aspect-ratio | `1-1`, `2-1`, `3-2`, `4-3`, `5-3`, `9-16`, `16-9` | N/A        |
+
+## Usage
+
+Here are examples of ratio atomics, used on `iframe`, `video` and `img` elements to scale elements. When used on a `video` element or `iframe` video with the matching aspect ratio, it will remove the black bars and scale responsively.
+
+```html
+  <div class="padding-sm width-500-desktop">
+  	<video controls name="media" class="width-500 aspect-ratio-16-9" src="~/src/scaffold/media/aspect-ratio-video-16-9.mp4" type="video/mp4" />
+  	</video>
+  </div>
+```
+
+```html
+<div class="width-250 padding-sm">
+	<iframe
+		id="embed-iframe"
+		class="aspect-ratio-9-16"
+		src="~/src/scaffold/media/aspect-ratio-video-9-16.mp4"
+		title="Vertical video player"
+		frameborder="0"
+		sandbox
+	></iframe>
+</div>
+```
+
+```html
+<div class="width-100 padding-xxs">
+	<img
+		class="aspect-ratio-1-1"
+		alt=""
+		role="presentation"
```

Comment:
Wondering if keeping the image around might be better for unsighted developers.

```
alt="An example image use to show an aspect ratio of 1 to 1."
```
then remove `role="presentation"`?

Reply:
done